src/EnergyPlus/WeatherManager.cc

```diff
@@ -4632,8 +4632,7 @@ Label9999: ;
 		}
 
 		if ( ! ErrorsFound ) {
-			StdBaroPress = 101.325 * std::pow( 1.0 - 2.25577e-05 * Elevation, 5.2559 );
-			StdBaroPress *= 1000.0;
+			StdBaroPress = StdBaroPressSeaLevel * std::pow( 1.0 - 2.25577e-05 * Elevation, 5.2559 );
```

Comment:
@EnergyArchmage, it is mathematically the same equation. It just does in one line now what was previously done in two. I can change it back, though, if you really want.